Makefile

```diff
@@ -0,0 +1,55 @@
+ORG=keycloak
+NAMESPACE=sso
+PROJECT=keycloak-operator
+REG=quay.io
+SHELL=/bin/bash
+TAG?=latest
+PKG=github.com/keycloak/keycloak-operator
+COMPILE_TARGET=./tmp/_output/bin/$(PROJECT)
+
+.PHONY: setup/mod
+setup/mod:
+	@echo Adding vendor directory
+	go mod vendor
+	@echo setup complete
+
+.PHONY: setup/travis
+setup/travis:
+	@echo Installing Operator SDK
+	@curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v0.10.0/operator-sdk-v0.10.0-x86_64-linux-gnu && chmod +x operator-sdk && sudo mv operator-sdk /usr/local/bin/
+
+.PHONY: code/run
+code/run:
+	@operator-sdk up local --namespace=${NAMESPACE}
+
+.PHONY: code/compile
+code/compile:
+	@GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o=$(COMPILE_TARGET) ./cmd/manager
+
+.PHONY: code/gen
+code/gen:
+	operator-sdk generate k8s
+
+.PHONY: code/check
+code/check:
+	@diff -u <(echo -n) <(gofmt -d `find . -type f -name '*.go' -not -path "./vendor/*"`)
+
+.PHONY: code/fix
+code/fix:
+	@gofmt -w `find . -type f -name '*.go' -not -path "./vendor/*"`
+
+.PHONY: image/build
+image/build: code/compile
+	@operator-sdk build ${REG}/${ORG}/${PROJECT}:${TAG}
+
+.PHONY: image/push
+image/push:
+	docker push ${REG}/${ORG}/${PROJECT}:${TAG}
```

Comment:
Travis will authenticate using docker login on its own, if running locally, I suppose the expectation would to be logged in already.

Reply:
@davidffrench I believe this can be removed, now that we agreed that quay.io will trigger the build.

@slaskawi just to give you more context: https://issues.jboss.org/browse/KEYCLOAK-11354

Reply:
Removed now